src/taxonomy/manage-orgs/ManageOrgsModal.scss

```diff
@@ -0,0 +1,7 @@
+.manage-orgs {
+  overflow: visible !important;
+
+  .pgn__modal-body {
+    overflow: visible;
```

Comment:
@rpenido Could you add the reason of this style in a commen?

Reply:
Done [9fbe76e](https://github.com/open-craft/frontend-app-course-authoring/pull/23/commits/9fbe76ebefb4ec86d57b0fce3d171f5bb84cddaa)!

Also updated the spacing here: [aca5ce5](https://github.com/open-craft/frontend-app-course-authoring/pull/23/commits/aca5ce5873580c50817e9404b27de54bfc71a744)

![image](https://github.com/open-craft/frontend-app-course-authoring/assets/849463/fbcebe3e-6887-44e2-92b2-aee7077f06ab)